shopinvader_api_payment_cart/routers/payment.py

```diff
@@ -0,0 +1,21 @@
+# Copyright Odoo SA (https://odoo.com)
+# Copyright 2024 ACSONE SA (https://acsone.eu).
+# @author Stéphane Bidoul <stephane.bidoul@acsone.eu>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+from odoo.fields import Command
+
+from odoo.addons.shopinvader_api_payment.schemas import TransactionCreate
+
+
+class ShopinvaderApiPaymentRouterHelper(models.AbstractModel):
+    _inherit = "shopinvader_api_payment.payment_router.helper"
+
+    def _get_payable_create_values(self, data: TransactionCreate) -> dict:
```

Comment:
```suggestion
    def _get_transaction_create_values(self, data: TransactionCreate) -> dict:
```

Reply:
Renamed `_get_additional_transaction_create_values`, because this method only adds additional parameters that will be returned as _kwargs_